contents/handbook/people/spending-money.md

```diff
@@ -73,7 +73,7 @@ PostHog will provide you with office equipment. Please note that it remains Post
 
 We'd prefer you to use a laptop. This is so when we host meetups in real life, you can easily bring your work with you. You should use an Apple laptop unless there is an exceptional reason why you can't, just to keep life simpler - for example, that means everyone can use the same software, and as we get bigger, it'll mean we're dealing with one supplier, not many. Please make sure you order a laptop with a keyboard that is US/UK/International English - this makes it much easier for us to reuse your laptop if you upgrade to another model or leave PostHog. 
 
-* For engineering roles, we recommend a Macbook Pro 14-inch M3 Pro, with the 11-core CPU, 14-core GPU upgrade and 32GB of RAM. By default grab the smallest SSD (512gb), but you can upgrade if you need to.
+* For engineering roles, we recommend a Macbook Pro 14-inch M4 Pro, with the 12-core CPU, 16-core GPU upgrade and 24GB of RAM. By default grab the smallest SSD (512gb), but you can upgrade if you need to.
```

Comment:
👎 for the RAM downgrade. I'd even suggest moving it the other way considering how the app is growing.

As a personal anecdote, my laptop (M1 Pro with 32GB) has been very sluggish for the last few months. It's been wapping a lot, and cosuming more than 32GB of RAM (around 8GB swapped most times). This may be due to PyCharm though. It got annoying enough that today for the first time I was only running vscode, to see if that makes a difference. My "memory used" is now at around 28GB and things are running better. However 28GB is still over 24GB. I just have our full stack running in vscode + chrome, music, notes, slack, other small stuff.

Please at least keep it at 32GB 🙏

Reply:
+1, seeing the same memory usage as Marius

Reply:
Gotcha! 32GB is not actually an option for the M4, next option would be 48GB @fraserhopper happy to go with this?

Reply:
There are ways to get a M4 Pro with 32GB. The configurator is a bit weird. My vote would still be to invest those extra $200 into the future and choose 48GB, as the app we're all working on is growing significantly, and we're already pushing really close to the 32GB limit (and over into slow-town if you give your development environment 8GB of memory instead of 4GB, as I had done).

Reply:
<img width="1072" alt="Screenshot 2024-11-17 at 11 36 22" src="https://github.com/user-attachments/assets/a3f53607-3f25-47b6-9767-6b33e9e04b68">
Another +1 for 24GB not being enough

Reply:
Yeah happy to go with the 48GB Ram for sure. It's the analogous upgrade here 
<img width="564" alt="Screenshot 2024-11-17 at 12 55 37" src="https://github.com/user-attachments/assets/dee99a88-b153-4808-bb6f-67891d30fa5f">